docs/user-manual/en/using-jms.md

```diff
@@ -170,9 +170,8 @@ Lastly, the `jgroups` scheme is supported which provides an alternative to the
 `udp` scheme for server discovery. The URL pattern is either
 `jgroups://channelName?file=jgroups-xml-conf-filename`
 where`jgroups-xml-conf-filename` refers to an XML file on the classpath that
-contains the JGroups configuration or it can be
-`jgroups://channelName?properties=some-jgroups-properties`. In both instance
-the `channelName` is the name given to the jgroups channel created.
+contains the JGroups configuration. The `channelName` is the name given to the
```

Comment:
The start of the earlier sentence also needs updated as it is no longer 'either'.